src/verifact_manager.py

```
@@ -76,16 +80,16 @@
             logger.info("Factchecking pipeline completed. Generated %d verdicts.", len(verdicts))
             return verdicts
 
-    async def _detect_claims(self, text: str) -> list[Claim]:
+    async def _detect_claims(self, text: str) -> List[Claim]:
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)